.github/workflows/build-container.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Build amc2moodle container'
+
+on:
+  push:
+    branches:
+      - 'master'
```

Comment:
Perhaps, running the docker on all push on master is too fast, for instance if CI, README files only are modified.
We could keep only the tag trigger.

Reply:
Done! (lines commented)